RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -361,22 +364,22 @@
 tobTecStep.inputClassifiers=['tobTecStepClassifier1','tobTecStepClassifier2']
 
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
-
-#LWTNN selector
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackingPhase1.toReplaceWith(tobTecStep, TrackLwtnnClassifier.clone(
+trackingPhase1.toReplaceWith(tobTecStep, tobTecStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorTobTecStep_Phase1'),
      src = 'tobTecStepTracks',
```

Comment:
this line is redundant and was not present in the pre-DNN version

Reply:
Removed redundant line